Src/zipkin4net.middleware.aspnetcore/Src/TracingMiddleware.cs

```diff
@@ -20,15 +23,18 @@ public static class TracingMiddleware
                 
                 var trace = traceContext == null ? Trace.Create() : Trace.CreateFromId(traceContext);
                 Trace.Current = trace;
-                using (var serverTrace = new ServerTrace(serviceName, getRpc(context)))
+                if (routeFilter(request.Path))
```

Comment:
Looking at the new changes with unit test (thanks for adding), I think you need to add `else` condition here too.

Reply:
this is the only blocking change, let's get this in an @bstauff publish!

Reply:
Sorry for the delay!  Got really busy.  I've made the changes, thanks for the catch!